sdk/include/ndk/halfuncs.h

```diff
@@ -152,20 +152,34 @@ HalEnableSystemInterrupt(
 );
 
 #ifdef __REACTOS__
+/* For APIC, the interrupt vector must also be transmitted. */
+typedef struct _HAL_INTERRUPT_CONTEXT
+{
+    PKTRAP_FRAME TrapFrame;
+    KIRQL Irql;
```

Comment:
Isn't that redundant with the `Irql` parameter of `HalEndSystemInterrupt()`?

Reply:
Removed